nodebuilder/share/constructors.go

```diff
@@ -52,3 +56,19 @@ func newModule(lc fx.Lifecycle, bServ blockservice.BlockService, avail share.Ava
 	})
 	return serv
 }
+
+// ensureEmptyCARExists adds the empty EDS from share.EnsureEmptySquareExists to the provided EDS store.
+func ensureEmptyCARExists(ctx context.Context, store *eds.Store) error {
+	bServ := blockservice.New(store.Blockstore(), nil)
+	// we ignore the error because we know that the batchAdder will not be able to commit to our
+	// Blockstore, which is not meant for writes.
+	eds, _ := share.EnsureEmptySquareExists(ctx, bServ)
```

Comment:
Feels a bit ugly. What if we make another helper in share that returns us an empty block EDS?

Reply:
done, using da.ExtendShares